Azure/azure-key-vault/ingest/parser.yml

```
@@ -0,0 +1,33 @@
+name: azure-key-vault
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: "database"
+          event.dataset: "keyvault"
+          event.outcome: "{{json_event.message.resultSignature}}"
+          event.action: "{{json_event.message.operationName}}"
+          event.type: ["access", "change", "info"]
+          cloud.provider: "azure"
```

I think we can add [`cloud.service.name`](https://www.elastic.co/guide/en/ecs/8.6/ecs-cloud.html#field-cloud-service-name):
```suggestion
          cloud.provider: "azure"
          cloud.service.name: "Azure Key Vault"
```

---

Should we also add
```
          observer.vendor: "Microsoft"
          observer.product: "Azure Key Vault"
```
?